template/script.sh.erb

```diff
@@ -47,24 +42,25 @@ EOL
 )
 chmod 700 "${RSESSION_WRAPPER_FILE}"
 
-# Define launcher for `rserver` if not using Singularity
-if [[ -z "${RSTUDIO_SINGULARITY_IMAGE}" ]]; then
-  PROOT_BIN="${PROOT_BIN:-"/usr/local/proot/bin/proot-x86_64"}"
-  RSERVER_LAUNCHER="${PROOT_BIN} -b $(mktemp -d):/tmp "
-fi
-
 # Set working directory to home directory
 cd "${HOME}"
 
 # Output debug info
 module list
 
+# Create tmpdir rooted in PBSTMP
+# This will be bound into the guest and mounted at /tmp
+CONTAINER_TMP=$(mktemp -d)
+
+set -x
 # Launch the RStudio Server
 echo "Starting up rserver..."
-set -x
-${RSERVER_LAUNCHER}rserver \
-  --www-port ${port} \
-  --auth-none 0 \
-  --auth-pam-helper-path "${RSTUDIO_SINGULARITY_HOST_MNT}${RSTUDIO_AUTH}" \
-  --auth-encrypt-password 0 \
-  --rsession-path "${RSTUDIO_SINGULARITY_HOST_MNT}${RSESSION_WRAPPER_FILE}"
+
+export SINGULARITY_BINDPATH="/usr/local,/etc/profile.d/lmod.sh,/usr/share/lmod,/opt/intel,/opt/mvapich2,/usr/lib64,/users,$CONTAINER_TMP:/tmp"
```

Comment:
If there was a module created to load the image, this would be perfect to put in that modulefile.